pallets/ethereum-transactions/src/ethereum_transaction.rs

```diff
@@ -325,6 +324,19 @@ impl EthAbiHelper {
         return ethabi::encode(&call.call_values)
     }
 
+    pub fn generate_confirmation_data_for_compacted_calls(
+        keccak_256_hash: &[u8; 32],
+        transaction_id: TransactionId,
+        from: &[u8; 32],
+    ) -> Vec<u8> {
+        let call_values: Vec<Token> = vec![
+            Token::FixedBytes(keccak_256_hash.to_vec()),
```

Comment:
Not sure if the root (in a publishRoot call) is a keccak hash. If so, I like this name. If not, it might be misleading, and I'd call it just `hash_256`.

Reply:
For now, this is for "compacted" calls and the parameter has a keccak_ prefix to make it obvious.
If/when we consume it in summary we should make it more generic and names more friendly.
Can do it now if you prefer that, and probably update the name as well